ExtraEnemyCustomization/Utils/RegexUtil.cs

```diff
@@ -1,13 +1,11 @@
 ﻿using System;
-using System.Collections.Generic;
-using System.Text;
 using System.Text.RegularExpressions;
 
 namespace EECustom.Utils
 {
     public static class RegexUtil
     {
-        public static readonly Regex VectorRegex = new Regex("[0-9.]+"); 
+        public static readonly Regex VectorRegex = new("-?[0-9.]+");
```

Comment:
Might prefer to keep this regex private? I don't see a use for it being public

Reply:
Fair enough, I'll move it to private